### ISSUE TITLE
StartStop trait didn't show stop in Smart Home App

### ISSUE THREAD

**Issue by jcyh0120** [actions-on-google/smart-home-nodejs#469]
I followed the instructions to setup an E2E demo of smart home.
I can do controls through the google assistant and google home app.

However, when using devices with Start/Stop trait, I cannot see the stop button after turned on the device.

How can I resolve this problem? Is there any setting missing?

**Comment by devunwired** [comment]
> I followed the instructions to setup an E2E demo of smart home.

Are you referring to this Node.js sample code, or another demo?

> However, when using devices with Start/Stop trait, I cannot see the stop button after turned on the device.

If you're experiencing an issue with how traits render in the Google Home app, I would recommend for you to file this issue in the [public issue tracker](https://issuetracker.google.com/issues/new?component=655104&template=1284148). You should also include the device type you are using (preferably a full SYNC response example).

**Comment by jcyh0120** [comment]
@devunwired Yes, I am using this nodejs sample code. Everything works fine except the StartStop render.

I bought a product with google smart home support and the render for stop button works fine in google home app. That's why I consider if there's anything else I should add in the code to let the stop button rendered.

Can anyone test if you could render the StartStop trait stop button in google home app with this nodejs demo?

I will use the public issue tracker to ask for help. Thanks!
